meson.build

```diff
@@ -0,0 +1,160 @@
+project(
+        'meson',
+	'c',
+	version: '3.0.0',
+	meson_version: '>=0.53.0',
+	default_options: [
+		'c_std=c99',
+		'warning_level=2',
+	],
+)
+
+add_project_arguments(
+	'-DSYSCONFDIR="@0@"'.format(get_option('sysconfdir')),
+	language: 'c'
+)
+
+if get_option('debug')
+	add_project_arguments('-DDEBUG', language: 'c')
+endif
+
+cc   = meson.get_compiler('c')
+os   = host_machine.system()
+conf = configuration_data()
+
+conf.set_quoted('COPYRIGHT', '(C) 2004-2018 Hisham Muhammad')
+conf.set_quoted('VERSION', meson.project_version())
+
+zfs_sources = []
+zfs_inc = ''
+
+math = cc.find_library('m', required: true)
```

Comment:
This should be `required: false` since `libm` is not used on all platforms, and on the platforms where it is used, it is always available.

https://mesonbuild.com/howtox.html#add-math-library-lm-portably

Reply:
Could you list which systems don't use libm?

Reply:
Nevermind, I get what you mean.